x/accounts/proto/cosmos/accounts/defaults/lockup/tx.proto

```diff
@@ -107,6 +107,17 @@ message MsgSend {
   ];
 }
 
+// MsgTrackUndelegation defines a message that enable lockup account to update delegation tracking
+message MsgTrackUndelegation {
```

Comment:
info: I was not sure what track means in this context.  I know naming is hard but how about something like: refresh undelegation state?

Reply:
maybe like update